cpp/CMakePresets.json

```diff
@@ -395,6 +421,16 @@
       "displayName": "Release build for PyArrow with everything enabled",
       "cacheVariables": {}
     },
+    {
+      "name": "ninja-release-emscripten-python",
```

Comment:
How about renaming this to `ninja-release-python-emscripten` because we have `ninja-release-python-maximal`?